src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -523,6 +525,39 @@ RequestContext<object> requestContext
             await requestContext.SendResult(result.First().ToString());
         }
 
+        private async Task HandleGetCommandRequest(
+            string param,
+            RequestContext<object> requestContext)
+        {
+            PSCommand psCommand = new PSCommand();
+            if (param != "") {
+                psCommand.AddCommand("Microsoft.PowerShell.Core\\Get-Command").AddArgument(param);
+            }
+            else
+            {
+                psCommand.AddScript("Microsoft.PowerShell.Core\\Get-Command -CommandType Function, Cmdlet, ExternalScript | Select-Object Name,ModuleName | Sort-Object Name");
```

Comment:
nit: AddScript can be evil. It's better to use AddCommand when possible

```csharp
psCommand.AddCommand("Microsoft.PowerShell.Core\\Get-Command")
    .AddParameter("CommandType", new[]{ "Function", "Cmdlet", "ExternalScript" })
    .AddCommand("Select-Object")
    .AddParameter("Property", new[] { "Name", "ModuleName" })
    .AddCommand("Sort-Object")
    .AddParamater("Property", "Name");
```

but haven't tested it 😄

Reply:
I've added this, it appears to be working, but I can't seem to attach the debugger at the moment to test... so I haven't fully tested this either 😜

Reply:
Able to attach debugger in vscode normal. This looks like I expected it to, and works too.